### PR TITLE
Added Seed Data

### PR DESCRIPTION
**Pull request by Juggernaut265**
The only real file that has been changed is "seeds.rb", which now creates five vendors and five products. Each vendor has an ownership of "Person of Color", and each product has a cert of "Halal". There are also placeholder pictures so that, when viewed locally, the products/vendors will appear on the carousel.

### CODE REVIEW

File: db/seeds.rb
Comment (nrleuenb) [inline db/seeds.rb]: We should create actual data to fill this, instead of leaving the fields empty; Adnan wanted the data so he could see offline, and we should have data to fill the details pages too. 

Maybe grab the same data that’s currently deployed from the Heroku version?

**Review by nrleuenb — CHANGES_REQUESTED** [review]

File: db/seeds.rb
Comment (Juggernaut265) [reply db/seeds.rb]: Added Some Data.

File: db/seeds.rb
Comment (nrleuenb) [inline db/seeds.rb:54]: Maybe have a slight variation in tags, so the different tags are all on at least one details page?

File: db/seeds.rb
Comment (nrleuenb) [inline db/seeds.rb]: I would add a Twitter or instagram link to at least one or two vendors, just to showcase the functionality.

**Review by nrleuenb — APPROVED** [review]
LGTM; maybe wait for Adnan’s response before merging though

**Pull request merged**
